python/common/org/python/types/Set.java

```diff
@@ -417,28 +417,52 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
 
     @org.python.Method(
             __doc__ = "Return the difference of two or more sets as a new set.\n\n(i.e. all elements that are in this set but not the others.)",
-            args = {"other"}
+            args = {"other"},
```

Comment:
@freakboy3742 , @deep110 I had used python 3.4 documentation. It seemed to suggest that it takes one args and rest arguments as varargs. Link to the documentation is below. Since the document with installing instructions said that voc uses python 3.4, I used python 3.4 documentation. I'll make the changes and make the code comply with 3.5. https://docs.python.org/3.4/library/stdtypes.html#frozenset